docs/gatsby-config.js

```diff
@@ -128,6 +128,7 @@ const themeOptions = {
               'host-collect-analyze/tcpPortStatus',
               'host-collect-analyze/udpPortStatus',
               'host-collect-analyze/time',
+              'host-collect-analyze/certificatesCollection',
```

Comment:
move this one up the list alphabetically 👍

Reply:
thank you! Changed